src/Agent.Worker/JobRunner.cs

```diff
@@ -218,6 +218,35 @@ public async Task<TaskResult> RunAsync(Pipelines.AgentJobRequestMessage message,
                     }
                 }
 
+                // Expand sidecar container properties
+                if (jobContext.SidecarContainers?.Count > 0)
```

Comment:
try avoid `?.`

Reply:
Done. SidecarContainers is always initialized now and I cleaned up call sites to be simpler and just use foreach